AspireStarterDb.ApiDbService/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using AspireStarterDb.ApiDbModel;
+using AspireStarterDb.ApiDbService;
+
+namespace Microsoft.Extensions.Hosting;
+
+internal static class ServiceCollectionExtensions
+{
+    public static IHostApplicationBuilder AddTodosDbContext(this IHostApplicationBuilder builder, string connectionName)
+    {
+        builder.Services.AddDbContextPool<TodosDbContext>(options =>
+        {
+            var connectionString = builder.Configuration.GetConnectionString(connectionName);
+
+            // Use this app's assembly for migrations, rather than the assembly containing the DbContext.
+            //options.UseNpgsql(connectionString, opt => opt.MigrationsAssembly(typeof(Program).Assembly.GetName()?.Name));
+            options.UseSqlServer(connectionString, opt => opt.MigrationsAssembly(typeof(Program).Assembly.GetName()?.Name));
```

Comment:
This should move to calling the overload that accepts the `Assembly` directly once dotnet/efcore#34772 is released.